build.gradle

```diff
@@ -98,22 +99,45 @@ configure(javaProjects) {
             dependency("org.opensaml:opensaml:2.6.4")
             dependency("org.python:jython-standalone:2.7.1")
             dependency("org.springframework.security.extensions:spring-security-saml2-core:1.0.2.RELEASE")
+            dependencySet(group: "com.google.protobuf", version: "3.4.0") {
```

Comment:
Horrible OCD nitpick on my part but I've tried to keep these in alphabetical order.